### PR TITLE
Various improvements to README.txt

### PR DESCRIPTION
**Pull request by luc-ass**
Just a small fixes to the README file: 
- `.js` ➡️ `.ts` to prevent dead links
-  typo
- added `goTo` values to the config table

_This is not a functional implementation. No new release is needed._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,14 +20,14 @@ For the underlying communication layer, it uses a port of the no-longer maintain
 ## Features
 
 - **Fan** as On-/Off-Switch. When switching off, directly back to the charging station.
-  - [Fanspeed levels](./models/speedmodes.js) adjustable via 3D Touch / Force Touch.
+  - [Fanspeed levels](./models/speedmodes.ts) adjustable via 3D Touch / Force Touch.
```
Comment (afharo) [inline README.md:23]: I think the link is still dead? Should it be?

```suggestion
  - [Fanspeed levels](./src/models/speedmodes.ts) adjustable via 3D Touch / Force Touch.
```

File: README.md [hunk 2]
```diff
@@ -20,14 +20,14 @@ For the underlying communication layer, it uses a port of the no-longer maintain
...
 - Battery status and condition in the device details. Low battery alert.
 - Pause switch (optional).
 - Room cleaning (optional): Read [Room cleaning](#room-cleaning) to understand how it works.
 - Zone cleaning (optional).
 - Occupancy sensor (similar to motion sensor) for dock status (optional).
 - Second Fan for water box modes (optional).
-  - [Watermode levels](./models/watermodes.js) only when enabled in config, and the device supports it.
+  - [Watermode levels](./models/watermodes.ts) only when enabled in config, and the device supports it.
```
Comment (afharo) [inline README.md:30]: Same as my previous comment applies
```suggestion
  - [Watermode levels](./src/models/watermodes.ts) only when enabled in config, and the device supports it.
```

File: README.md [hunk 3]
```diff
@@ -107,24 +107,28 @@ For the underlying communication layer, it uses a port of the no-longer maintain
...
+| `goToWord`            | `"go to coordiantes"` | used for autonaming the "go to" command. E.g: Hey Siri, NameOfYourVacuum to to coordinates                                                                                                                                                                                                                                                                                                                                                                                                  |
+| `goToX`               | `"where are you"`     | x-coordinate of the specified location                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```
Comment (afharo) [inline README.md:120]: I believe the default value is 25000?

File: README.md [hunk 4]
```diff
@@ -176,14 +180,14 @@ To use this plugin, you have to read the "token" of the xiaomi vacuum robots. He
...
-NOTE: We are not currently aware of how to retrieve the token from the Roborock App. Please, share any findings in the issue #104.
+NOTE: We are not currently aware of how to retrieve the token from the Roborock App. Please, share any findings in the issue [#104](https://github.com/homebridge-xiaomi-roborock-vacuum/homebridge-xiaomi-roborock-vacuum/issues/104).
 
 ## Is my model supported?
 
 As new users join our community, and use this plugin, we try to learn from their device, and keep up with the new devices Xiaomi releases. However, the main developer @afharo, only owns the model S5, and he can't test the features on other models. If you wonder whether your device is supported, please:
 
-1. Check the [./models/models.js](./models/models.js) file to see if your device is already in the explicit list.
-2. If not, maybe it matches de _default_ behaviour (just try running the plugin and see if it works).
+1. Check the [./models/models.ts](./models/models.ts) file to see if your device is already in the explicit list.
```
Comment (afharo) [inline README.md:189]: Same as the previous comments. We need to nest it into src
```suggestion
1. Check the [models.ts](./src/models/models.ts) file to see if your device is already in the explicit list.
```

**Review by afharo — CHANGES_REQUESTED** [review]
Thank you for the changes!! It's great to see folks care about the readme 🧡

I think the links need to be updated and, afaik, the default values of the goToX and goToY are not correct. Other than that, it looks great!

File: README.md (hunk 3 above)
Comment (luc-ass) [reply README.md]: 25500. corrected it.

**Comment by luc-ass** [comment]
No idea why the checks fail…

**Comment by luc-ass** [comment]
here we go

**Review by afharo — APPROVED** [review]
@luc-ass you rock!!

**Pull request merged**